package.json

```diff
@@ -80,7 +80,6 @@
     "grunt-newer": "^1.1.1",
     "grunt-ng-annotate": "^1.0.1",
     "grunt-ng-constant": "^2.0.1",
-    "grunt-node-inspector": "^0.4.1",
```

Comment:
It was the first thing I had to do to get grunt serve to run so I just committed it. Can revert if necessary